python/guillotina/config.yaml

```diff
@@ -0,0 +1,12 @@
+framework:
+  website: guillotina.io
+  version: 6.0
+
```

Comment:
you need to add 
```yaml
build_deps:
  - libffi-dev
```
since `argon2-cffi` requires this library (not bny default on alpine)

Reply:
you also need 
```yaml
fixes:
  - curl https://sh.rustup.rs > init.sh
  - sh init.sh -y
  - find $HOME/.cargo/bin -type f -exec install {} /usr/local/bin \;
  - rustup default nightly
```

since `rust` is used to build `orjson`

Reply:
@masipcat time to continue this `PR` ?